src/locales/en.json

```diff
@@ -18,7 +18,17 @@
           "line": {
             "1": "Your Cozy will be removed",
             "2": "Your will not be able to access your Cozy anymore",
-            "3": "All your data will be removed"
+            "3": "All your data will be removed",
+            "4": "Please enter your password to validate this action:"
+          }
+        },
+        "password": {
+          "hide": "Hide",
+          "show": "Show",
+          "required": "This field is required.",
```

Comment:
nit: Why do we end this phrase with a `.` but we don't do it for the other errors?